nilearn/connectome/connectivity_matrices.py

```diff
@@ -193,9 +193,10 @@ def _geometric_mean(matrices, init=None, max_iter=10, tol=1e-7):
     return gmean
 
 
-def sym_to_vec(symmetric):
-    """Return the flattened lower triangular part of an array, after
-    multiplying above the diagonal elements by sqrt(2).
+def sym_to_vec(symmetric, discard_diagonal=False, isometric=True):
```

Comment:
I must admit that now that only diagonal elements are modified, I don't see the usage. Anybody in favor of keeping this option?